chain-abci/src/main.rs

```diff
@@ -41,6 +41,7 @@ struct AbciOpt {
     )]
     host: IpAddr,
     #[structopt(
+        default_value = "",
```

Comment:
Instead of providing a default blank string. Can we make `genesis_app_hash: Option<String>`?

Reply:
ok